libobs/audio-monitoring/pulse/pulseaudio-wrapper.h

```diff
@@ -29,8 +29,8 @@ struct pulseaudio_default_output {
 
 struct enum_cb {
 	obs_enum_audio_device_cb cb;
-	void *data;
-	int cont;
+	void *                   data;
```

Comment:
Pointer does not align with name.

Reply:
Probably will disable all alignment options due to this, they don't seem to add to much to the actual readability of the code.